sql/moz-fx-data-shared-prod/subscription_platform_derived/daily_active_logical_subscriptions_v1/backfill.yaml

```diff
@@ -1,3 +1,15 @@
+2025-05-22:
+  start_date: 2021-09-09
+  end_date: 2025-05-14
```

Comment:
The end dates are further in the past because as of today these ETLs are delayed by an extra 7 days to allow for late-arriving data (#7471), meaning the next nightly run will target 2025-05-15, so we only have to backfill before that.

Reply:
I didn't realize managed backfills currently take the `date_partition_offset` setting into account, so with this ETL having `date_partition_offset: -7` this actually should have been
```yaml
  start_date: 2021-09-16
  end_date: 2025-05-21
```
However, I have submitted https://github.com/mozilla/bigquery-etl/pull/7510 to add an `ignore_date_partition_offset` backfill option, which would allow using the more natural start/end dates directly matching the table partitions to be backfilled.

Reply:
And now #7522 enabled the new `ignore_date_partition_offset` option for these backfills.